### PR TITLE
Callback function `on_write`

### PR DESCRIPTION
**Pull request by MarcMush**
here is the second version of a callback function after each page is built and written

```julia
function my_function(pg, jd_vars)
  # whatever you want
end
# ...
serve(...; on_write=my_function)
# ...
```
_Originally posted by @tlienart in https://github.com/tlienart/JuDoc.jl/pull/309#issuecomment-566483523_

I had to add it at multiple places (that's why I went for the simpler solution in #309) but this way it's probably more user-friendly

### CODE REVIEW

**Comment by tlienart** [comment]
Thanks a lot Mark, I'll have a detailed look asap

File: src/manager/judoc.jl [hunk 1]
```diff
@@ -23,7 +23,7 @@ Keyword arguments:
 * `no_fail_prerender=true`: whether, in a prerendering phase, ignore errors and try to produce an output
 * `eval_all=false`:  whether to force re-evaluation of all code blocks
 * `silent=false`:    switch this on to suppress all output (including eval statements).
-* `cleanup=true`:    whether to clear environment dictionatires, see [`cleanup`](@ref).
+* `cleanup=true`:    whether to clear environment dictionaries, see [`cleanup`](@ref).
 """
```
Comment (tlienart) [inline src/manager/judoc.jl]: Could you please add a short  entry for `on_write` here?

**Comment by tlienart** [comment]
Thanks a lot for these changes! I'll incorporate them in the coming PR #349

**Comment by MarcMush** [comment]
sorry for the delay
hopefully I didn't make any mistake while changing to franklin

**Comment by tlienart** [comment]
this  is now #350, thanks a  lot!

**Pull request closed without merging**